particles/Arcs/source/Noop.js

```diff
@@ -0,0 +1,17 @@
+// @license
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+'use strict';
+
+defineParticle(({DomParticle}) => {
```

Comment:
is this particle used anywhere? i'm not finding it :(

Reply:
No, but I used it while working on this stuff. I think it's useful to have available.